applications/examples/example_number_input/scenes/example_number_input_scene_input_number.c

```diff
@@ -0,0 +1,48 @@
+#include "../example_number_input_i.h"
+
+void example_number_input_scene_input_number_callback(void* context) {
+    ExampleNumberInput* app = context;
+    view_dispatcher_send_custom_event(
+        app->view_dispatcher, ExampleNumberInputCustomEventTextInput);
+}
+
+void example_number_input_scene_input_number_on_enter(void* context) {
+    furi_assert(context);
+    ExampleNumberInput* app = context;
+    NumberInput* number_input = app->number_input;
+    size_t enter_name_length = 5;
+    char* str = "Enter a number (0 - 9999)";
```

Comment:
Numbers are clearly off

Reply:
The option is that you can set your own text in your app. But I will adjust this, so that the text transferred is dynamic matching the min/max numbers.